platform_channels/android/app/src/main/kotlin/dev/flutter/platform_channels/AccelerometerStreamHandler.kt

```diff
@@ -29,9 +29,12 @@ class AccelerometerStreamHandler(sManager: SensorManager, s: Sensor) : EventChan
     override fun onAccuracyChanged(sensor: Sensor?, accuracy: Int) {}
 
     override fun onSensorChanged(sensorEvent: SensorEvent?) {
-        if (sensorEvent != null) {
+        if (sensorEvent == null) {
+            throw Exception()
+        }else{
```

Comment:
```suggestion
        } else {
```
nit: formatting is not proper.

Reply:
Sure thing!